tests/run/embedsignatures.pyx

```diff
@@ -185,7 +188,7 @@ __doc__ = ur"""
     f_defexpr4(int x=(Ext.CONST1 + FLAG1) * Ext.CONST2)
 
     >>> print(funcdoc(f_defexpr5))
-    f_defexpr5(int x=4)
+    f_defexpr5(int x=2 + 2)
```

Comment:
Well, I guess this is not a blocker for this PR. We can decide later on it, right?